core/templates/dev/head/pages/library-page/search-bar/search-bar.directive.ts

```diff
@@ -32,7 +32,9 @@ angular.module('oppia').directive('searchBar', [
     return {
       restrict: 'E',
       scope: {},
-      bindToController: {},
+      bindToController: {
+        dropup: '&'
```

Comment:
Is there a reason why it's named this way? (it sounds weird, but I'm okay if it's from convention).

Reply:
Actually, it was for flipping the dropdown list in the search bar to a dropup list, as in the initial screenshot. I'll rename it to make it clearer.